pkgs/shells/rc/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv, fetchurl, autoreconfHook
+, ncurses #acinclude.m4 wants headers for tgetent().
+, historySupport ? false
+, editSupport ? "readline", readline ? null, libedit ? null
+}:
+
+assert editSupport == "readline" -> readline != null;
```

Comment:
I've went ahead with the null-able _readline_. But mind you, this doesn't solve the problem; Only postpones it.

But I think it's both a Nix issue and a NixPkgs one: Some incorrectly ```assert gtk2Support || gtk3Support || qtSupport;``` whereby a 1, 0, 1 wouldn't raise an exception. Others who realize the issue, avoid asserting conditions by cutting out features or simply not putting them in...

I believe a language construct is in order. I already have a good idea in mind for it I think... We'll see :)